.travis.yml

```diff
@@ -5,6 +5,7 @@ services:
 before_install:
   - mkdir -p _output/
   - sudo apt-get install -y git
+  - sudo apt-get install libpcap-dev
```

Comment:
it will install libpcap on host system, you need it in build container (check run_tests part with build-image)